lib/services/elasticsearch.js

```diff
@@ -81,6 +81,29 @@ module.exports = function (kuzzle, options) {
   };
 
   /**
+   * Return the list of documents matching the ids given in the body param
+   * NB: Due to internal Kuzzle mechanism, can only be called on a single index/collection,
+   * using the body { ids: [.. } syntax.
+   * @param {RequestObject} requestObject
+   * @returns {Promise}
+   */
+  this.mget = function (requestObject) {
```

Comment:
Nice!
I've logged a new jira ticket (KUZ-298) to implement a new API route using this method.